modules/data_team_helper/php/NDB_Form_data_team_helper.class.inc

```diff
@@ -286,12 +346,16 @@ class NDB_Form_Data_Team_Helper extends NDB_Form
      * @param unknown_type $test_name   the value of test name
      * @param string       $visit_label the value of visit label
      * @param string       $candidate   the value of candidate
+     * @param string       $site        the value of site
+     * @param string       $project     the value of project
```

Comment:
are you sure these are strings? It looks like they're being in the where column for SQL columns that are integers, not strings.

Can you also make the description more descriptive? What is "the value of site"? Is it the id? Is it the name? The abbreviation?